downloads/swan-lake-release-notes/swan-lake-preview5/RELEASE_NOTE.md

```diff
@@ -0,0 +1,256 @@
+
+```ballerina
+type Coordinates record {|
+    int latitude;
+    int longitude;
+|};
+
+isolated class Hotels {
+
+    private map<Coordinates> hotels = {};
+
+    isolated function add(string name, Coordinates coordinates) {
+        lock {
+            self.hotels[name] = coordinates.clone();
+        }
+    }
+
+    isolated function getCoordinates() returns map<Coordinates> {
+        lock {
+            return self.hotels.clone();
+        }
+    }
+}
+```
+
+An `isolated` function can also access a `final` variable if its type is a subtype of `isolated object {}`.
+
+```ballerina
+final Hotels hotels = new;
+
+isolated function getCoordinates() returns Coordinates[] => hotels.getCoordinates().toArray();
+```
+
+##### Removed XML functional constructors
+
+XML functional constructor syntax has been removed. In place of XML functional constructors, `'xml:createXXX` functions should be used. 
+
+**Disallowed syntax**
+```ballerina
+'xml:Element element = 'xml:Element("elem");
+'xml:Comment comment = 'xml:Comment("comment content");
+'xml:ProcessingInstruction proceInstruc = 'xml:ProcessingInstruction("DONOT", "print this");
+'xml:Text text = 'xml:Text("this is a character sequence");
+```
+
+**Valid syntax**
+```ballerina
+'xml:Element element = 'xml:createElement("elem");
+'xml:Comment comment = 'xml:createComment("comment content");
+'xml:ProcessingInstruction proceInstruc = 'xml:createProcessingInstruction("DONOT", "print this");
+'xml:Text text = 'xml:createText("this is a character sequence");
+```
+
+##### Unified object method scope and object field scope 
+
+Object field scope and object method scope are unified into a single object scope. This means that it's no longer possible to have fields and methods with the same name.
+
+With this change, object methods can be accessed as object fields and the reference to `self` is captured in the resulting function pointer.
+
+```ballerina
+class Accumulator {
+    int i;
+
+    function init(int i) {
+        self.i = i;
+    }
+
+    function accumulate(int acc) {
+        self.i = self.i + acc;
+    }
+
+    function get() returns int {
+        return self.i;
+    }
+}
+
+Accumulator accumulator = new(0);
+var acc = accumulator.accumulate; // access method as a field
+var getVal = accumulator.get;
+
+acc(1); // similar to calling accumulator.accumulate(1);
+acc(2);
+int val = getVal();
+```
+
+##### Improved lang library functions
+
+Several changes and improvements have been introduced to the `ballerina/lang.value` lang library module.
+
+###### New `ensureType` function
+
+A new `ensureType` function has been introduced to convert any kind of a Ballerina value to a user-specified type.
```

Comment:
Shall we change this line as follow, 
```md
##### `ensureType` langlib method

This function can be used to check a value that belongs to the given type; similar to type cast expression. But this langlib method returns an error value if the type test fails. 
```

@MaryamZi @praneesha Please check.

Reply:
Fixed.